conf/roslin.config

```diff
@@ -40,12 +34,12 @@ process {
     // This is very useful any JAVA programs which allocate task.memory RAM for its Virtual Machine
     // Also it leaves enough memory for singularity to unpack images.
     withName: '!.*FASTQC.*' {
-        // Check if an environment variable NFX_SGE_PROJECT exists, if yes, use the stored value for -P option
-        // Otherwise set the project to uoe_baseline
-        if (System.getenv('NFX_SGE_PROJECT')) {
-            clusterOptions = {"-l h=!node1d01 -l h_vmem=${(task.memory + 8.GB).bytes/task.cpus} -P $NFX_SGE_PROJECT"}
-        } else {
-            clusterOptions = {"-l h=!node1d01 -l h_vmem=${(task.memory + 8.GB).bytes/task.cpus} -P uoe_baseline"}
+        clusterOptions = {
```

Comment:
Process specific configs should not go into this config, they should go in pipeline specific ones.

Otherwise pipelines that don't use FASTQC will get ugly warnings saying 'this process doesn't not exist'

Reply:
This was setup as a workaround of the [memory overallocation of FASTQC](https://github.com/nf-core/modules/pull/6156). The fix seems to be integrated to the pipelines now. I can remove it then. 
Thanks for the suggestion.